scripts/createConfig.js

```diff
@@ -112,24 +124,36 @@ networks.forEach(network => {
     cacheConfig.storage.staking.contractAddress = storageConfig[network].staking;
 
     // Pinning Service
-    const storagePinningConfig = require("./templates/storagePinning.json");
+    const pinnerNetworkName = network.startsWith('rsk') ? network.substr('rsk'.length) : network;
```

Comment:
What do you mean `'rsk'.length`? Isn't that just 3? Or were you meaning to refactor the `'rsk'` to a variable?
Or was it just so that it is clear where the substring number 3 comes from?

Reply:
it's the last one